bot.py

```diff
@@ -84,11 +84,30 @@ async def daily_reminder(ctx, time):
 @client.command(aliases=["start-pomodoro"])
 async def pomodoro(ctx, time):
     # Set a timer for 25 minutes
+    time = 1500
     # Notify the user after 25 minutes
+    message = await ctx.send("Your countdown has been started for 25 minutes")
+    while True:
+        time -= 1
+        if time == 0:
+            await message.edit(content=("Time remaining {:02d}:{:02d}".format(0, 0)))
+            await ctx.send(ctx.message.author.mention + " your countdown has ended!\nBreak starts.")
+            break
+
+        await asyncio.sleep(1)
     # Set the timer for 5 minutes (starting of the break)
-    # Notify the user after 5 minutes that Pomodoro has ended
-    pass
+    time = 300
+    await message.edit(content=("Your break countdown has been started for 5 minutes"))
+    while True:
+        time -= 1
+        if time == 0:
+            await message.edit(content=("Time remaining {:02d}:{:02d}".format(0, 0)))
+            await ctx.send(ctx.message.author.mention + " your break countdown has ended!")
```

Comment:
Your Break has ended is repetitive, because we notify the user that their Pomodoro has ended.